### ISSUE TITLE
[BUG] Shop not destroyable as player even if he is the region owner

### ISSUE THREAD

**Issue by Plattio** [snowgears/shopbugs#287]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Type of bug

Other unexpected behaviour

### Environment

```markdown
- Server Version: 1.18 Paper (Git: b475c6a)
- Shop Version: 1.8.3.3
```

### Server startup log

secret

### Shop config files

https://gist.github.com/Plattio/ffb7cf71a8639e531e07e1bbf2c0955a

### Bug description

You can't destroy the shop of other players even if you are the WorldGuard region owner. (Also tried it with WorldGuardHook true)
If i give the permission shop.destroy.other they can destroy every shop.

### Steps to reproduce

1. Create a buy/sell shop sign
2. Try to destroy it with your alt-account (with shop.destroy permission)

### Expected behaviour

WorldGuard Owner/Member can destroy the shop of other players. (Because we have a shopping mall and the owner is mostly changing after 7 days)

### Actual behaviour

WorldGuard Owner can't destroy the shop sign

**Comment by snowgears** [comment]
Fixed in 1.8.3.4 (in progress)

**Comment by snowgears** [comment]
"I'm on a world without regions, so i've created with my 1th account a shopchest. Then i created around this shopchest a region and added my 2th account as owner. My 2th account still can't destroy the shopchest from the 1th account, but the 2th account is the region owner.

Or is there any new config option for this fix? Around the shopchest region is just the global region without any flags."

**Comment by snowgears** [comment]
Confirmed to be fixed in 1.8.3.6 (in progress)
